grama/eval_defaults.py

```diff
@@ -240,7 +360,8 @@ def eval_conservative(model, quantiles=None, df_det=None, append=True, skip=Fals
             of values for each random variable, or None for default 0.01.
             values in [0, 0.5]
         df_det (DataFrame): Deterministic levels for evaluation; use "nom"
```

Comment:
The `(DataFrame)` type helper should be changed to `(DataFrame or None)`

Reply:
Great points! Will update all of these functions.